content/posts/weekly/July-2022.md

```diff
@@ -16,3 +16,18 @@ weight: 60
 - On a Bodhi authentication error, Packit will retry the task multiple times
   in ten-minute intervals to be able to fix the issue in the meantime.
   ([packit-service#1564](https://github.com/packit/packit-service/pull/1564))
+
+## Week 28 (July 12th–July 18th)
+
+- Packit now guides everyone better when a FAS account is private
+  and updates the internal information about installations correctly
+  when the app is reinstalled.
+  ([packit-service#1575](https://github.com/packit/packit-service/pull/1575))
+- Packit can now correctly create Bodhi updates using the new Bodhi 6 client.
+  ([packit#1651](https://github.com/packit/packit/pull/1651))
+- RPM build commands of Packit CLI have been merged into one build subcommand,
+  for more information see the updated [documentation](https://packit.dev/docs/cli/build/).
+  We have also introduced a new --srpm option to the new build subcommand
+  that can be used to trigger local, Copr or Koji build from an already built
+  SRPM rather than the one implicitly created by Packit.
+  ([packit#1611](https://github.com/packit/packit/pull/1611))
```

Comment:
Even though this is CLI-specific, I think it's okay to have it here for the change to get visibility.